blocks/editable/format-toolbar.js

```diff
@@ -151,17 +152,24 @@ class FormatToolbar extends Component {
 						className="editable-format-toolbar__link-modal"
 						style={ linkStyle }
 						onSubmit={ this.submitLink }>
+						<label
```

Comment:
Is there any difference between a visually-hidden `label` for screen reader purposes, vs. using the `aria-label` attribute on the input itself?

@afercia Would you have insight here?